Source/qol/stash.cpp

```diff
@@ -542,6 +543,19 @@ void StashStruct::RemoveStashItem(uint16_t iv)
 	Stash.dirty = true;
 }
 
+void StashStruct::SetPage(int newPage)
+{
+	page = clamp(newPage, 0, CountStashPages - 1);
+}
+
+void StashStruct::RefreshItemStatFlags()
+{
+	// Iterate through active page and update if a item is usable

```

Comment:
Was tempted to suggest changing this loop but you were a step ahead :D. Only thing I'd be hesitant about is if potential slowdown when opening the stash if someone ends up with more than a few hundred items in the stash. It's a pretty fast operation anyway so I don't think this'll be noticeable even with a full stash.

```suggestion
```
The doxygen comment on the method is more accurate than this inline comment, no value keeping this one.

Reply:
Gone 😉